time stamp
string    command # "write", "read", "reboot", "disable", "enable", "show"
uint8[]   id_list

string    write_read_parameter
float64[] write_value_list